app/models/journal_manager.rb

```diff
@@ -254,4 +259,15 @@ def self.create_custom_field_data(journable, journal)
       journal.customizable_journals.build custom_field_id: cv.custom_field_id, value: cv.value
     end
   end
+
+  # Wrap send_notification so it defaults to true, when it's nil
+  def send_notification
+    return true if @send_notification.nil?
+    @send_notification
+  end
```

Comment:
Isn't this method useless?

Looks like:

* it is defined on the instance rather than on the class
* the class version of this method is already defined in line 32
* I don't see why on the world `@send_notification` should ever be `nil`